### PR TITLE
Feat: added question page of quiz app

### PR DESCRIPTION
**Pull request by vivekmehto**
image

### CODE REVIEW

File: routes/questions.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      href="/assets/favicon/gambit-favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="https://gambitui.netlify.app/gambit.min.css" />
```
Comment (krishanu2911) [inline routes/questions.html]: just a suggestion to move this link to your styles base CSS so then you just need to import only one link

**Review by krishanu2911 — APPROVED** [review]
Nice design, no changes required well-written code. Added just a small suggestion

**Review by raman1999 — APPROVED** [review]
Well written code,you can merge it

File: routes/questions.html (hunk 1 above)
Comment (vivekmehto) [reply routes/questions.html]: yes, I will change it. Thank you

**Pull request merged**